### DISCUSSION TITLE
inaccessible examples

### DISCUSSION THREAD

**Discussion by Tyrant-sudo** [byuflowlab/FLOWUnsteady#93] [General]
The example results' link at https://edoalvar2.groups.et.byu.net/public/FLOWUnsteady/ seems to be inaccessible. I am unable to view the visualization results. I wonder if there is an updated link or another way for me to access these results?

**Comment by Tyrant-sudo** [comment]
There is also the more practical question of what do I do if I want to get the effect of the hub when dealing with coaxial rotors？

**Comment by EdoAlvarezR** [comment]
You are actually not supposed to have access to [edoalvar2.groups.et.byu.net/public/FLOWUnsteady](https://edoalvar2.groups.et.byu.net/public/FLOWUnsteady/) ¯\\\_(ツ)\_/¯

The links at the end of the tutorial seem to be working well though: [LINK](https://flow.byu.edu/FLOWUnsteady/visualization/).

**Reply by Tyrant-sudo** [reply]
Thank you very much. However, I have encountered another question. I have configured a pair of counter-rotating coaxial rotors, but the results indicate that the upstream rotor is achieving higher thrust. How can I properly configure it?
